lintreview/tools/checkstyle.py

```diff
@@ -56,10 +63,24 @@ def process_files(self, files):
 
         process_checkstyle(self.problems, output, docker.strip_base)
 
-    def create_command(self, files):
+    def setup_properties(self, properties_file):
+        config_loc = os.path.dirname(docker.apply_base(self.options['config']))
+        project_loc = docker.apply_base('/')
+
+        properties = {
+            'config_loc': config_loc,
+            'samedir': config_loc,
+            'project_loc': project_loc,
+            'basedir': project_loc,
+        }
+
+        jprops.store_properties(properties_file, properties)
```

Comment:
This is much improved over having to serialize properties files in the checkstyle bridge.